docs/API.md

```diff
@@ -166,12 +166,36 @@ curl -v \
 "https://profile.accounts.firefox.com/v1/avatar"
 ```
 
-#### Response
+
+#### Response 
+
+> Note: we no longer support Gravatar uploading, but there are entries in the database
```

Comment:
Are there?  Side note, but I wonder if we should somehow purge them.

Reply:
Mistake, we actually used to process them, I guess the docs always had an incorrect url